test/kuttl/tests/glance_single/01-assert.yaml

```diff
@@ -149,8 +149,8 @@ kind: TestAssert
 namespaced: true
 commands:
   - script: |
-      template='{{.status.apiEndpoint.internal}}{{":"}}{{.status.apiEndpoint.public}}{{"\n"}}'
-      regex="http:\/\/glance-internal.$NAMESPACE.*:http:\/\/glance-public.$NAMESPACE.*"
+      template='{{ index .status.apiEndpoint "default-internal" }}{{ ":" }}{{ index .status.apiEndpoint "default-public" }}'
```

Comment:
I see the following error locally: 

```
error: error executing template "{{ index .status.apiEndpoint \"default-internal\" }}{{ \":\" }}{{ index .status.apiEndpoint \"default-public\" }}": template: output:1:3: executing "output" at <index .status.apiEndpoint "default-internal">: error calling index: index of untyped nil
```
when the `NAMESPACE` is not set.
Let's see the CI execution, but we might need to properly set it!

Reply:
I see the same error in #530, and I have to investigate how the `namespace` is passed to these scripts.

Reply:
@gibizer if you rebase #530 on this I think we can see if we still get this error ^

Reply:
Done